psycop/common/model_training_v2/trainer/data/data_filters/geography.py

```diff
@@ -0,0 +1,75 @@
+from collections.abc import Sequence
+from typing import Literal
+
+import polars as pl
+
+from psycop.common.model_training_v2.config.baseline_registry import BaselineRegistry
+from psycop.common.model_training_v2.trainer.base_dataloader import BaselineDataLoader
+from psycop.common.model_training_v2.trainer.data.data_filters.base_data_filter import (
+    BaselineDataFilter,
+)
+from psycop.common.model_training_v2.trainer.data.data_filters.geographical_split.make_geographical_split import (
+    get_regional_split_df,
+)
+
+
+@BaselineRegistry.data_filters.register("regional_data_filter")
+class RegionalFilter(BaselineDataFilter):
+    def __init__(
+        self,
+        regions_to_keep: Sequence[Literal["vest", "midt", "øst"]],
+        id_col_name: str = "dw_ek_borger",
+        timestamp_col_name: str = "timestamp",
+        regional_move_df: pl.LazyFrame | None = None,
+        region_col_name: str = "region",
+        timestamp_cutoff_col_name: str = "first_regional_move_timestamp",
+    ):
+        """Filter data to only include ids from the desired regions. Removes
+        predictions times after the patient has moved to a different region to
+        avoid target leakage. If regional_move_df is None, the standard regional
+        move dataframe is loaded .
+
+        Args:
+            regions_to_keep (Collection[Literal["vest", "midt", "øst"]]): The regions to keep
```

Comment:
Remove types from docstring
```suggestion
            regions_to_keep: The regions to keep
```